slowmovie.py

```diff
@@ -234,6 +237,7 @@ def error(self, message):
 parser.add_argument("-l", "--loop", action="store_true", help="loop a single video; otherwise play through the files in the videos directory")
 parser.add_argument("-e", "--epd", help="the name of the display device driver to use")
 parser.add_argument("-o", "--loglevel", default="INFO", type=str.upper, choices=["DEBUG", "INFO", "WARNING", "ERROR", "CRITICAL"], help="minimum importance-level of messages displayed and saved to the logfile (default: %(default)s)")
+parser.add_argument("-C", "--clear", action="store_true", help="clears displays when exithandler is triggered")
```

Comment:
```suggestion
parser.add_argument("-C", "--clear", action="store_true", help="clear display on exit")
```
A little clearer, I think.